.pipelines/.vsts-garabge-collection.yaml

```diff
@@ -0,0 +1,24 @@
+pool:
```

Comment:
I don't see triggers. Do you plan to run it manually for some time?

Reply:
we'll add a schedule to the pipeline definition itself